src/panels/config/entity_registry/ha-config-entity-registry.ts

```diff
@@ -60,58 +77,129 @@ class HaConfigEntityRegistry extends LitElement {
           filterable: true,
           direction: "asc",
         },
-        entity_id: {
-          title: this.hass.localize(
-            "ui.panel.config.entity_registry.picker.headers.entity_id"
-          ),
-          sortable: true,
-          filterable: true,
-        },
-        platform: {
-          title: this.hass.localize(
-            "ui.panel.config.entity_registry.picker.headers.integration"
-          ),
-          sortable: true,
-          filterable: true,
-          template: (platform) =>
-            html`
-              ${this.hass.localize(`component.${platform}.config.title`) ||
-                platform}
-            `,
-        },
-        disabled_by: {
-          title: this.hass.localize(
-            "ui.panel.config.entity_registry.picker.headers.enabled"
-          ),
-          type: "icon",
-          template: (disabledBy) => html`
-            <ha-icon
-              slot="item-icon"
-              .icon=${disabledBy ? "hass:cancel" : "hass:check-circle"}
-            ></ha-icon>
+      };
+
+      const statusColumn: DataTableColumnData = {
+        title: this.hass.localize(
+          "ui.panel.config.entity_registry.picker.headers.status"
+        ),
+        type: "icon",
+        sortable: true,
+        filterable: true,
+        template: (_status, entity: any) => html`
```

Comment:
Can't we remove this wrapping html tag since it's all just a turnarary statement ?